osu.Game/Overlays/BeatmapSet/Buttons/PreviewButton.cs

```diff
@@ -74,6 +74,13 @@ private void load(OsuColour colours, OverlayColourProvider colourProvider)
             background.Colour = colourProvider.Background6;
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            playButton.Enabled.BindValueChanged(e => Enabled.Value = e.NewValue, true);
```

Comment:
well, that's the design of `PreviewButton`, it creates a `PlayButton` inside of it, `PlayButton` is the one managing over the beatmap set's preview track, `PreviewButton` only adds to the look of the button. don't think it's feasible to change the direction of propagation without rewriting `PreviewButton` all over.

Reply:
no, that's not quite the same. the preview track and the playing bindable are re-exposed in `PreviewButton`. it's composition. if it were possible to proxy `Enabled`, i'd recommend that, but that doesn't seem feasible as the preview button already has its own `Enabled`. so it'd have to be declared with `new` which i don't even know if it'll behave correctly since shadowing is confusing.

speaking of which, given this direction of propagation, how do you even disable a `PreviewButton`? setting `Enabled` on the preview doesn't propagate to the play button, and you can't disable the inner button from outside because it's private.

Reply:
All good points, does this commit satisfy what you wanted https://github.com/ppy/osu/commit/1734a889b838023ca310bfc3fd5378dc1f22fce2, otherwise not sure I can come up with anything else simple to do, helping would be appreciated.

Reply:
The direction you have on that commit seems.. better? see comments i left on it.